backend/migrations/006-455-favorite.sql

```diff
@@ -0,0 +1,24 @@
+BEGIN;
```

Comment:
This is missing two things:
- the create table is missing so this won't work on an existing DB which is the purpose
- it needs a check to see if the table already exists, if it does it shouldn't try to re-create everything because that generates a bunch of errors. The migration script should be re-runnable without errors.